courses/models.py

```diff
@@ -0,0 +1,52 @@
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+from uuidfield import UUIDField
+
+class Course(models.Model):
+    """
+    An edX course which is purchaseable.
+    """
+    uuid = UUIDField(auto=True)
+    title = models.CharField(max_length=255)
+    author_name = models.CharField(max_length=255)
+    overview = models.TextField()
+    description = models.TextField()
+    video_url = models.URLField()
+
+    edx_instance = models.CharField(max_length=255)
+    instructors = models.ManyToManyField(User)
+    live = models.BooleanField(default=False)
+
+    price_per_seat_cents = models.IntegerField(null=True, blank=True, help_text="Cost of the whole course per seat in cents")
+
+    deleted = models.BooleanField(default=False)
```

Comment:
+1  I agree that systems that manage financial transactions don't delete anything.  Items are just flagged as deleted.  Otherwise you can't perform an audit.  :-(